modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,33 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling the Metering Operator from {product-title}
 
-You can remove metering from your cluster.
+The following guide provides details on removing the Metering Operator and its components.
 
-.Prerequisites
+[NOTE]
+====
+Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
+====
```

Comment:
I moved this note further up in the document as it felt out of place mentioning this towards the bottom.